tests/talker_test.py

```diff
@@ -0,0 +1,7 @@
+from __future__ import annotations
+
+import comictaggerlib.ui.talkeruigenerator
+
+
+def test_format_internal_name():
+    assert comictaggerlib.ui.talkeruigenerator.format_internal_name("talker_comicvine_cv_test_name") == "Test name"
```

Comment:
parametrize this. See https://docs.pytest.org/en/6.2.x/parametrize.html

add "talker_comic_vine_cv2_test_name" as another test case (because someone is going to put an underscore in their talker id and there is no reason to try to stop them)

Reply:
I've switch to using the `dest` value from the config; `cv_test_name`, that does still assume the first block of text is a single string. I'm not sure of another way to format those strings without making that assumption. The other option could be to add something like a `display` option to `settngs` but I'm not sure it would make sense for anyone else?

It may also not be a good idea to expect `dest`?

Reply:
using `cv_test_name` or `cv test name` would be acceptable as that is what the commandline option will be `--cv-test-name` and `cv_test_name` is what is directly used in the config file. Adding a display name might be a good idea as I can make it default to `dest` if it is not set. I might make that change later.
Until I make that change I want at least 3 test cases on this function, even if it seems trivial and all that this function does is `name.replace("_", " ").title()`

`dest` is always set and it is what is used internally to identify the options when it is a dictionary, so it is a good option to use it here.

Reply:
I've added another test case with the same assumption. If you do add a `display` option to `settngs` then obviously the function and test isn't needed any more.

Reply:
I see you've added `display_name` so I've switched to using that.